lib/mollie/api/util.rb

```diff
@@ -0,0 +1,37 @@
+module Util
+  extend self
+
+  def nested_underscore_keys(hash)
```

Comment:
I'm not really sure if I like this. I understand that in ruby we want snake_case instead of CamelCase properties.  But maybe it's cleaner to add a method to the `Resource` classes to map the API response to the `Object` classes?

What do you think?